@here/harp-mapview/lib/Tile.ts

```diff
@@ -607,11 +608,12 @@ export class Tile implements CachedResource {
         this.m_elevationRange.minElevation = elevationRange.minElevation;
         this.m_elevationRange.maxElevation = elevationRange.maxElevation;
         this.m_elevationRange.calculationStatus = elevationRange.calculationStatus;
+        this.elevateGeoBox();
 
         // Only elevate bounding box if tile has already been decoded and a maximum geometry height
```

Comment:
Maybe better to say "update" bounding box, because it is the name of the method call.

Reply:
done